data/korath/korath missions.txt

```diff
@@ -257,3 +270,207 @@ mission "Small Scale Delivery"
+					goto defenses
+				`	"An afterburner and fuel, of course."`
+					goto afterburner
+				`	"Power and cooling would be wonderful."`
+					goto coolpower
+				`	"I'll take the pair of fiery turrets."`
+					goto inferno
+				`	"Doomsday torpedoes? Count me in!"`
+					goto torpedoes
+
+			# This is the default option because you can't go wrong with defenses.
+			# A mix of outfit types and sizes lets you outfit a wide variety of
+			# designs. It's easy to pillage any of these, so this is the wimpiest
+			# option, but also the most versatile.
+			label defenses
+			action
+				outfit "Systems Core (Medium)"
+				outfit "Systems Core (Small)"
+				outfit "Warder Anti-Missile" 2
+				outfit "Small Heat Shunt"
+				outfit "Large Heat Shunt"
+				outfit "Fuel Processor"
+			`	"Done! We're also giving you <payment>. These outfits will keep you alive through tough fights."`
+				goto end
+
+			# You need serious power and cooling to support two Infernos,
+			# so the gift is smaller than it appears.
+			label inferno
+			action
+				outfit "Korath Inferno" 2
+			`	"Done! We're also giving you <payment>. You'll need some hefty cooling and a lot of weapon space to support these turrets."`
+				goto end
+
+			# This much power and cooling opens up a lot of designs, but
+			# primarily on larger ships. If you don't want outfits, this
+			# is the best option since it sells for 3 million credits.
+			label coolpower
+			action
+				outfit "Double Plasma Core"
+				outfit "Large Heat Shunt" 3
+				outfit "Small Heat Shunt"
+			`	"Done! We're also giving you <payment>. Only the largest human and Hai vessels can make use of this generator."`
+				goto end
+
+			# Do you want a capital ship that can outrun drones?
+			label afterburner
+			action
+				outfit "Afterburner (Planetary Class)"
+				outfit "Fuel Processor" 3
+			`	"Done! We're also giving you <payment>. Make sure you have enough cooling and engine space to support this afterburner."`
+				goto end
+
+			# This is a secret for players to find. It is the only time
+			# in the game where you can get Firestorm Torpedo Racks.
+			# Anything else you can pillage, so by some measure this is
+			# the "right" option.
+			label torpedoes
+			action
+				outfit "Firestorm Torpedo Rack" 2
+				outfit "Firestorm Torpedo" 39
+				outfit "Firestorm Battery"
+			`	"Done! We're also giving you <payment>. Be careful with these torpedoes. They have a wide blast radius. They'll cripple your ship and escorts if they detonate too close."`
+				goto end
+
+			label end
+			`	"It is always a pleasure to help another warrior get their due."`
+
+	npc kill
+		government Korath
+		system Almach
+		personality staying timid target marked
+		fleet "Korath Stealth Relay"
+		dialog "With the probes destroyed, you can now return to Stormhold for your payment."
+	npc
+		government Korath
+		system Almach
+		personality staying heroic marked opportunistic
+		fleet
+			variant
+				"Ra'gru Ik 618 (Ground Assault)"
```

Comment:
A "Ground Assault" variant doesn't seem like the most appropriate name for a ship on guard duty to protect some drones. Is there a reason why the Korath would use a ship calibrated for Ground Assault missions as a sentry here?

As per the comments on the ship definition, I'm suggesting using a more appropriate (Sentry) label here instead of the (Ground Assault), since the latter suggests a ship that has different outfitting for a ground invasion, whereas the former simply states what it is:, a Sentry.

```suggestion
				"Ra'gru Ik 618 (Sentry)"
```